test/production/eslint/test/next-build-and-lint.test.ts

```diff
@@ -18,6 +18,9 @@ describe('Next Build', () => {
           dependencies: {
             // create-next-install will replace this with a version built from the local source
             'eslint-config-next': 'canary',
+            // TODO(jiwon): update this test when we create the config for ESLint 9.
+            // pin ESLint 8 for current testing as we don't create the corresponding config for the users yet.
```

Comment:
Does a v8 config not work in v9?

Reply:
The skipped test is `"first time setup"` test, which it tests creating a config `.eslintrc.json` for the user and runs correctly on the next run. I skipped it because I did not add logic of creating appropriate config based on the version (since the goal is to unblock, not full-support).

Reply:
FYI The tests for v9 is at https://github.com/vercel/next.js/pull/71229

Reply:
> The skipped test is "first time setup" test, which it tests creating a config .eslintrc.json for the user and runs correctly on the next run. I skipped it because I did not add logic of creating appropriate config based on the version (since the goal is to unblock, not full-support).

So v9 does not support v8 configs?

Reply:
Looks like we could do it by `ESLINT_USE_FLAT_CONFIG=false`.
https://eslint.org/docs/latest/use/migrate-to-9.0.0#-new-default-config-format-eslintconfigjs

Will try, thank you!